packages/cursorless-vscode-e2e/src/suite/fixtures/scopes/javascript/function.scope

```diff
@@ -0,0 +1,25 @@
+function myFunk() {
+
+}
+---
+
+[Content]
+[-------------------
+ function myFunk() {
+ 
+ }
+ -]
+
+[Removal]
+[-------------------
+ function myFunk() {
+ 
+ }
+ -]
+
+[Domain]
+[-------------------
+ function myFunk() {
+ 
+ }
+ -]
```

Comment:
as with removal, I wonder if we just leave out domain if it's equal to content

Reply:
Done